target/linux/ath79/dts/ar9344_netgear_wndr.dtsi

```diff
@@ -171,22 +171,25 @@
 			reg = <0x3c0000 0x300000>;
 		};
 
-		kernel@6c0000 {
-			label = "kernel";
-			reg = <0x6c0000 0x400000>;
-		};
-
-		ubiconcat0: partition@ac0000 {
-			label = "ubiconcat0";
-			reg = <0xac0000 0x1500000>;
-		};
-
 		partition@6c0000 {
 			label = "firmware";
 			reg = <0x6c0000 0x1900000>;
-			compatible = "openwrt,uimage", "denx,uimage";
+			compatible = "openwrt,uimage", "denx,uimage", "fixed-partitions";
 			openwrt,ih-magic = <0x33373033>;
 			openwrt,ih-type = <IH_TYPE_FILESYSTEM>;
+
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			kernel@6c0000 {
```

Comment:
`kernel@0`

Reply:
Looks like I forgot to recompile. Will fix.